lib/apiScenario/postmanCollectionRunnerClient.ts

```diff
@@ -909,20 +951,27 @@ try {
     step: string,
     prevMethod: string = "put",
     scriptTypes: PostmanHelper.TestScriptType[] = [],
-    armTemplate?: ArmTemplate
+    armTemplate?: ArmTemplate,
+    finalStateVia?: string
   ): Item {
     const { item } = this.addNewItem(
       "Blank",
       {
         name: `_${name}_final_get`,
         request: {
           method: "GET",
-          url: "",
+          url: `{{x_final_get_url}}`,
         },
       },
       baseUri
     );
-    item.request.url = url;
+
+    /**
+     * set for original uri
+     */
+    if (finalStateVia === "original-uri" || (prevMethod === "put" && !finalStateVia)) {
+      item.request.url = url;
```

Comment:
can you correlate the initial request and final response, and feed into schema validator? The final response body must match swagger definition

Reply:
Corresponding ARM RPC rule: https://github.com/Azure/oav/issues/932